### ISSUE TITLE
No base step listener registered when trying to run the tests from feature file

### ISSUE THREAD

**Issue by therodrigocosta** [serenity-bdd/serenity-cucumber4#20]
Hi,

I've create a project with Serenity and cucumber 4, however, I'm not being able yet to run the tests from feature file (using IntelliJ idea) as I used to do with cucumber 2.
I can run the tests with I try direct from the CucumberTestSuite runner, but from feature file it isnt happening.
Is there an issue with serenity and cucumber 4 runner? Or am I doing something wrong?

This is the configuration of my CucumberTestRunner and the IntelliJ configuration:
![image](https://user-images.githubusercontent.com/40844089/70548782-e8304580-1b51-11ea-8940-b4a3723a548f.png)

Those are the versions that I'm using on the project
`        <serenity.cucumber.version>1.0.15</serenity.cucumber.version>
        <cucumber.version>4.2.0</cucumber.version>
        <webdriver.driver></webdriver.driver>
        <serenity.maven.version>2.0.45</serenity.maven.version>`

I've tried to change versions with many different combinations, however it did not work also.

Does anyone have already had this issue? 

Thanks!

**Comment by wakaleo** [comment]
I don't think this is supported for Cucumber 4 (I didn't think it still worked for Cucumber 2 as there was an IntelliJ issue, which I presume they have fixed).

**Comment by therodrigocosta** [comment]
Hi @wakaleo thank you very much!

I've follow the steps described on your blog
https://johnfergusonsmart.com/running-cucumber-serenity-feature-files-directly-intellij/
It used to work for me with cucumber 2.

As this is an IntelliJ issue I'll report it for them.


Thank you very much!

**Comment by therodrigocosta** [comment]
And just to provide a litlle bit more of details, these are the error messages that I'm getting

```
[main] WARN cucumber.runtime.SerenityBackend - It looks like you are running a feature using @RunWith(Cucumber.class) instead of @RunWith(CucumberWithSerenity.class). Are you sure this is what you meant to do?
[main] ERROR net.thucydides.core.steps.StepEventBus - No base step listener registered - this is generally a bad sign.
```
